applications/serial_lte_modem/src/http_c/slm_at_httpc.c

```diff
@@ -572,6 +574,11 @@ int handle_at_httpc_request(enum at_cmd_type cmd_type)
 
 	switch (cmd_type) {
 	case AT_CMD_TYPE_SET_COMMAND:
+		if (httpc.state > HTTPC_INIT && httpc.state < HTTPC_END) {
```

Comment:
The state stays as HTTPC_INIT until response_cb is called. If during this time, there is another HTTPCREQ coming, then it will bypass this check and overwrite the data_buf.

Reply:
I think that @SeppoTakalo:s suggestion of having HTTP_ONGOING would go well with this, so that the HTTP_ONGOING would be set in handle_at_httpc_request after checking that HTTP is not currently going on.